zuul.d/jobs.yaml

```diff
@@ -1,3 +1,10 @@
+- job:
+    name: atmosphere-unit-test
+    parent: golang-go
+    vars:
+      go_version: 1.21.13
+      go_command: test ./... -v
```

Comment:
Can you split this into two jobs?

One called `atmosphere-golang-go` with the `go_version` variable in there.  Then, another one called `atmosphere-golang-go-test` which the parent of the is `atmosphere-golang-go` which has the test there.  :)

Reply:
well, it means the first one should be an abstract?

Reply:
That's correct. :)